src/net/RTP/RTPSession.cs

```diff
@@ -436,6 +451,15 @@ public bool HasVideo
             m_bindPort = bindPort;
 
             m_sdpSessionID = Crypto.GetRandomInt(SDP_SESSIONID_LENGTH).ToString();
+
+            if (IsSecure)
+            {
+                m_srtpHandler = new SrtpHandler();
+                
+                SrtpCryptoSuites = new List<SDPSecurityDescription.CryptoSuites>();
+                SrtpCryptoSuites.Add(SDPSecurityDescription.CryptoSuites.AES_CM_128_HMAC_SHA1_80);
+                SrtpCryptoSuites.Add(SDPSecurityDescription.CryptoSuites.AES_CM_128_HMAC_SHA1_32);
+            }
         }

```

Comment:
WebRTC connections set `IsSecure` but don't need to use the `SrtpHandler`. Would it be better to create this object when the remote SDP offer/answer is received so that it can be distinguished between a VoIP+SRTP and WebRTC connection?